<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>上传文件</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="8" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QPushButton" name="upload">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>黑体</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>上传</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="Exit2">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
              <property name="text">
               <string>退出</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="6" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <family>黑体</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>服务器ip</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="uploadServerIP">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>50</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <family>黑体</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>本地ip</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="uploadLocalIP">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>50</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="FileChoose">
              <property name="font">
               <font>
                <family>黑体</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>选择文件</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="FilePath">
              <property name="minimumSize">
               <size>
                <width>400</width>
                <height>50</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="7" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <family>黑体</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>传输模式</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="uploadMode">
              <property name="minimumSize">
               <size>
                <width>125</width>
                <height>50</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>黑体</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <item>
               <property name="text">
                <string>netascii</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>octet</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>下载文件</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>服务器ip</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="downloadServerIP">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>服务器文件名称</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="downloadServerFilename">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>本地ip</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="downloadLocalIP">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>本地文件名称</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="downloadLocalFilename">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="4" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>传输模式</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="downloadMode">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>netascii</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>octet</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item row="5" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QPushButton" name="download">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
              <property name="text">
               <string>下载</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="Exit1">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
              <property name="text">
               <string>退出</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="ClearButton">
          <property name="text">
           <string>清除内容</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="output">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
